packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -597,8 +605,8 @@ export class GarbageCollector implements IGarbageCollector {
             return baseGCDetailsMap;
         });
 
-        // Initialize the base state. The base GC data is used to detect and log when inactive / deleted objects are
-        // used in the container.
+        // Initialize the base state that is used to detect when inactive objects are used. It is explicitly initialized
+        // for non-summarizer clients as they don't run GC. Summarizer clients do this when GC runs first time.
         if (this.shouldRunGC) {
             this.initializeBaseStateP.catch((error) => {
                 const dpe = DataProcessingError.wrapIfUnrecognized(
```

Comment:
Tangential questions about this code:

* What kind of errors do we see here?  Are they originating from our code or partner container/DataStore implementations?
* Should we put other props on the error like invalid timeout?

Reply:
The only error that can happen here is if the base summary that the container loads from does not have correct GC state.
What is invalid timeout in this context?

Reply:
Doh my brain I mean inactive timeout.  And just any other props on the class's logger - Tony and I have talked about adding a method to `ChildLogger` to dump all props so they can be added to an error in a case like this, would that be useful?

Reply:
Yes, I think its worth adding as much as info as possible.
I am actually thinking of adding a log when GC initializes that dumps all these settings. A lot of time when I look at telemetry, I want to know what was enabled for a particular client and what wasn't. If we add that we don't need to add any of these here right?
Also, do you think adding such a telemetry on every GC load is too much?

Reply:
Added the inactive timeout to this one for now.